crates/oxc_ast/src/ast/jsx.rs

```diff
@@ -23,9 +23,9 @@ use super::{inherit_variants, js::*, literal::*, ts::*};
 #[ast(visit)]
 #[derive(Debug, Hash)]
 #[cfg_attr(feature = "serialize", derive(Serialize, Tsify))]
```

Comment:
Can't we also put these in `#[ast]`? Similar to how we are adding ast derive.

Reply:
To clarify I'm talking about this line:

```rust
#[cfg_attr(feature = "serialize", derive(Serialize, Tsify))]
```

Reply:
You mean add `#[cfg_attr(feature = "serialize", derive(Serialize, Tsify))]` in `#[ast]` macro so we can remove it from type defs?

If that's what you mean, I think this would be a more "controversial" change, because it hides away things from the reader ("How does this type get serialized? It doesn't implement `Serialize`!"). In comparison, the changes in this PR don't really do that - they just strip out some largely meaningless syntax noise. Pretty sure Boshen would be against going so far as hiding `#[derive]`s.